packages/Tabs/src/components/Tab/Tab.styles.js

```diff
@@ -56,7 +56,7 @@ const horizontalStyles = ({ tabHeight }) => css`
   display: inline-flex;
   height: ${getHeightValue(tabHeight)};
   margin-right: ${tokens.space};
-  padding: ${stylers.spacer(2)} ${tokens.space} ${tokens.space} ${tokens.space};
+  padding: ${tokens.space};
```

Comment:
Sorry, I looked a little close at the [UI Kit](https://www.figma.com/file/DRhxHNWVzd94K8Tf0C3VTy/%F0%9F%90%A6Starling-UI-Kit?node-id=1737%3A39455) and actually since there's a 4px bottom border, this still isn't quite right.  

In fact, with the `align-items: center` it doesn't really need extra vertical padding, _except_ to balance out with that bottom border.  

I _think_ what you want is `4px 8px 0 8px`.  But I haven't verified that.  (Also, there's a `tokens.spaceSm` for `4px`.